experiments/adding-new-experiments/add-new-subject.sh

```diff
@@ -0,0 +1,170 @@
+# The root directory for the old subject
+export DT_SUBJ_ROOT=/home/user/dependent-tests-impact/experiments/ignite-old/modules/hadoop
+# Directory for where all of the old subject's information is stored.
+export DT_SUBJ=$DT_SUBJ_ROOT/target
+
+# The root directory for the new subject
+export NEW_DT_SUBJ_ROOT=/home/user/dependent-tests-impact/experiments/ignite-new/modules/hadoop
+# Directory for where all of the new subject's information is stored.
+export NEW_DT_SUBJ=$NEW_DT_SUBJ_ROOT/target
+
+# The name of the subject you wish to add (e.g., Zeppelin-Server) 
+export SUBJ_NAME=hadoop
+
+#FILL ABOVE FIRST
+
+# Directory for all the jar files containing the DT-impact tools and its dependences
+export DT_TOOLS=/home/user/dependent-tests-impact/experiments/impact-tools/*
+
+# Directory for libraries of the old subject
+export DT_LIBS=$DT_SUBJ/dependency/*
+# Compiled class files of the old subject
+export DT_CLASS=$DT_SUBJ/classes
+# Compiled automatically-generated test files of the old subject
+export DT_RANDOOP=$DT_SUBJ/randoop/bin
+# Compiled human-written test files of the old subject
+export DT_TESTS=$DT_SUBJ/test-classes
+
+# Directory for libraries of the new subject
+export NEW_DT_LIBS=$NEW_DT_SUBJ/dependency/*
+# Compiled class files of the new subject
+export NEW_DT_CLASS=$NEW_DT_SUBJ/classes
+# Compiled automatically-generated test files of the new subject
+export NEW_DT_RANDOOP=$NEW_DT_SUBJ/randoop/bin
+
+# Directory for all the jar files containing the DT-impact tools and its dependences
+export PRIO_RESULTS=/home/user/dependent-tests-impact/experiments/prioritization-results
+export SELE_RESULTS=/home/user/dependent-tests-impact/experiments/selection-results
+export PARA_RESULTS=/home/user/dependent-tests-impact/experiments/parallelization-results
+
+# Directory for all the jar files containing the DT-impact tools and its dependences
+export PRIO_DT_LISTS=/home/user/dependent-tests-impact/experiments/prioritization-dt-list
+export SELE_DT_LISTS=/home/user/dependent-tests-impact/experiments/selection-dt-list
+export PARA_DT_LISTS=/home/user/dependent-tests-impact/experiments/parallelization-dt-list
+
+#for passing these into prioritization-runner.sh, selection-runner.sh, etc 
+export DT_TOOLS_DIR=/home/user/dependent-tests-impact/experiments/impact-tools
+export DT_LIBS_DIR=$DT_SUBJ/dependency
+export NEW_DT_LIBS_DIR=$NEW_DT_SUBJ/dependency
+
+
+
+
+#PRIORITIZATION
```

Comment:
Is there a reason why the comments below are kept? If not, please delete these extra lines.